CorsixTH/Lua/dialogs/watch.lua

```diff
@@ -65,6 +69,9 @@ function UIWatch:UIWatch(ui, count_type)
     self.end_button = self:addPanel(end_sprite, 4, 0)
       :makeButton(4, 0, 27, 28, end_sprite + 1, self.onCountdownEnd)
       :setTooltip(tooltips[count_type])
+  elseif count_type == "emergency" then
```

Comment:
Just else, or probably better move this condition up and have the currently ~= "emergency" condition as the else clause.

Reply:
Actually I think it'd be better to do this emergency clause for tick_rate up where tick_rate is defined, instead of in this block which is deciding the end_button.

Reply:
I'll rejig